app/endpoints/conversations.py

```
@@ -0,0 +1,38 @@
+from app.schemas.conversation import ConversationCreate, Conversation
+from app.crud.conversation import create_conversation, get_conversation, get_conversations, update_conversation, delete_conversation
+from app.core.database import get_db
+
+router = APIRouter()
+
+# @router.post("/", response_model=Conversation)
+# def create_new_conversation(conversation: ConversationCreate, db: Session = Depends(get_db)):
+#     return create_conversation(db=db, conversation=conversation)
+
+@router.get("/", response_model=list[Conversation])
+def read_conversations(skip: int = 0, limit: int = 10, db: Session = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)